test/Microsoft.Azure.Devices.Edge.Test/PriorityQueues.cs

```diff
@@ -73,105 +148,109 @@ public async Task PriorityQueueModuleToModuleMessages()
                                {
                                    ["TestReportType"] = "CountingReport",
                                    ["TestOperationResultType"] = "Messages",
-                                   ["ExpectedSource"] = $"{loadGenModuleName}.send",
-                                   ["ActualSource"] = $"{relayerModuleName}.receive",
+                                   ["ExpectedSource"] = $"{LoadGenModuleName}.send",
+                                   ["ActualSource"] = $"{RelayerModuleName}.receive",
                                    ["TestDescription"] = "unnecessary"
                                }
                            }
                        });
 
-                    builder.AddModule(loadGenModuleName, loadGenImage)
+                    builder.AddModule(LoadGenModuleName, loadGenImage)
                         .WithEnvironment(new[]
                         {
-                            ("testResultCoordinatorUrl", trcUrl),
+                            ("testResultCoordinatorUrl", TrcUrl),
                             ("senderType", "PriorityMessageSender"),
                             ("trackingId", trackingId),
-                            ("testDuration", loadGenTestDuration),
+                            ("testDuration", LoadGenTestDuration),
                             ("messageFrequency", "00:00:00.5"),
-                            ("priorities", priorityString)
+                            ("priorities", string.Join(';', testInfo.Priorities)),
+                            ("ttls", string.Join(';', testInfo.Ttls)),
+                            ("ttlThresholdSecs", "100")
                         });
 
-                    Dictionary<string, object> routes = this.BuildRoutes(priorityString.Split(';'), loadGenModuleName, relayerModuleName);
+                    Dictionary<string, object> routes = this.BuildRoutes(testInfo.Priorities, testInfo.Ttls, LoadGenModuleName, RelayerModuleName);
                     builder.GetModule(ModuleName.EdgeHub).WithDesiredProperties(new Dictionary<string, object> { ["routes"] = routes });
                 });
+        }
 
-            EdgeDeployment deployment = await this.runtime.DeployConfigurationAsync(addInitialConfig, token);
-            PriorityQueueTestStatus loadGenTestStatus = await this.PollUntilFinishedAsync(loadGenModuleName, token);
+        private Dictionary<string, object> BuildRoutes(List<int> priorities, List<int> ttls, string sendModule, string receiveModule)
```

Comment:
Uses TestInfo instead
```suggestion
        private Dictionary<string, object> BuildRoutes(TestInfo testInfo, string sendModule, string receiveModule)
```